server/SessionStorage.js

```diff
@@ -7,16 +7,47 @@ export default class SessionStorage {
 
     createSession(data) {
         let key;
+        let date = +new Date();
         do {
             key = crypto.randomBytes(256).toString('hex');
         } while (key in this.storage);
 
-        this.storage[key] = data;
-
+        this.storage[key] = { 
+            data
```

Comment:
Why not do: `this.storage[key] = { data, date }`?

or even: `this.storage[key] = { data, accessedAt: +new Date() }`?
and remove line 10 too.

`accessedAt` is a much better name than `date`, because it conveys the meaning of the variable.